docs/guides/session-management/low-latency-session-management.mdx

```diff
@@ -0,0 +1,113 @@
+
+```bash
+curl -o /dev/null \
+     -w %{time_connect}:%{time_starttransfer}:%{time_total} \
+     -s https://playground.projects.oryapis.com/sessions/whoami \
+     -H 'ory_session_playground=MTY1MjcyMjk5OXxBenA5MVdvaXpPZ242QlZtOVNCZ2JuRWt4NVB4WFFIVGdLcmFIZl9BN0VTOHZ3VHlJbVdVenpaZl96VWtWS1p4clI3bFRkYnFBeDZwdFZGU3RvSmxyY0JpUGJLNGQwLUlfWnNCMlFlMS1CbDlBdEtjX0xoLWxmSUNmWUJWU24xWTBoNlBGM0laSHc9PXyOu8_5YP6pFxhuNLWD3fmzxdo8JWo27iy0NkF2HQ7fcA=='
+
+0.030575:0.265693:0.265756
+```
+
+That basic example above shows:
+
+1. 0.030575 seconds spent connecting to the playground.
+1. 0.265693 seconds passed to start transferring data.
+1. 0.265756 total seconds spent on the request.
+
+
+## How to validate session and when do they invalidate
+
+You can validate a session using the following options:
+
+1. By using `toSession` method of Ory SDK
+1. By calling `/sessions/whoami` endpoint
+
+A session invalidates in these cases:
+
+1. When the user logs out;
+1. When it expires;
+1. When one wants to force a user to logout by calling [revokeSession](https://github.com/ory/kratos-client-go/blob/master/docs/V0alpha2Api.md#RevokeSession).
+
+## How to cache sessions
+
+Caching allows reusing previously retrieved or computed data efficiently. A cache is a high-speed data storage
+layer that stores a subset of data, typically transient. Future requests for that data are served up faster than
+is possible by accessing the data’s primary storage location.
+
+One of the most used strategies is to use the least recently used (LRU) cache. There are many options to use in a system:
+
+1. Embedded in-memory caches such as LevelDB, BerkeleyDB, Flashdb, etc.
+1. Using Key-Value storage such as Redis, Memcached, Aerospike cache, etc.
+
+Depending on requirements, one can choose any of the key-value storage available but consider this:
+
+1. Using embedded in-memory solutions can give one low latency. However, an application stores state can prevent horizontally scaling it.
+1. Key-value storages add network latency, but an application does not store any state so that one can scale it without any issues.
+
+:::warning
+
+Be careful with the TTL for cached sessions, and do not forget to remove the session once it expires or a user logs out. Users can still have access because of cached data, leading to a potential security breach.
+
+:::
+
+Let us take a look at following examples. We have session middleware that validates sessions and caches data. The logic is simple
+
+1. Try to get cached session from Redis;
+1. If a session is valid, return it and do not request Ory Cloud;
+1. If data does not persist, validate a session against Ory Cloud and cache it.
+
+## Network architecture
+
+1. `8080` port is used by the REST API
+1. `4000` port is used by [`ory proxy`](https://ory.sh/docs/guides/cli/proxy-and-tunnel) that manages cookies for local domain and proxies traffic to Ory Cloud
```

Comment:
As someone who isn't really familiar with the topic and your train of thought, I don't understand what's this section doing here. What's the purpose of talking about the proxy?